schema/spacewalk/common/tables/suseCredentials.sql

```diff
@@ -28,9 +28,14 @@ CREATE TABLE suseCredentials
     username VARCHAR(64) NOT NULL,
     password VARCHAR(4096) NOT NULL,
     created  TIMESTAMPTZ DEFAULT (current_timestamp) NOT NULL,
-    modified TIMESTAMPTZ DEFAULT (current_timestamp) NOT NULL
-)
+    modified TIMESTAMPTZ DEFAULT (current_timestamp) NOT NULL,
+    extra_auth bytea,
+    payg_ssh_data_id NUMERIC
+        CONSTRAINT suse_credentials_payg_ssh_data_id_fk
+            REFERENCES susePaygSshData (id)
+);
 
-;
+ALTER TABLE susecredentials
+    ADD CONSTRAINT suse_credentials_payg_ssh_data_id_uq unique (payg_ssh_data_id);
```

Comment:
I will check it, but I think it can cope with several different entries with null values. 
But I will change this and add a condition to the unique constraint.

Reply:
I just checked it and null can be used multiple times.
In docs [1] says: "Null values are not considered equal. "

However, if you think that having a partial unique index would be better, even for readability, I'm fine on changing it. 

[1] https://www.postgresql.org/docs/13/indexes-unique.html

Reply:
If it also works this way i am fine